0000-node-requirements/0000-node-requirements.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Interledger Node - Requirements Specification
+draft: 1
+---
+# Interledger Node - Requirements Specification
+
+This document defines the basic functions of an ILP node. 
+
+An ILP node is a system that performs the necessary functions to route ILP packets between peers on the open Interledger network.
+
+The document is specific to the requirements of a "core" node that only operates at the ILP layer and does not concern itself with functionality in the higher protocol layers (e.g. STREAM, PSK2, etc). In other words, this document describes the functions of a node that is neither a sender or receiver of ILP packets but rather a "middlebox", to borrow some Internet terminology.
+
+Such a node is often referred to as a **connector** as it provides connectivity between different networks.
```

Comment:
Then why not call this document "Interledger Connector - Requirements"?

If you are going to call it node then it should also encompass non-routing nodes, i.e. senders and receivers.

Reply:
Fair point